repositories/repositories.bzl

```diff
@@ -23,18 +23,19 @@ load(
     "http_archive",
     "http_file",
 )
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+load("@bazel_toolchains//deps:io_bazel_rules_docker.bzl", io_bazel_rules_docker_version = "version")
 
 def repositories():
     """Download dependencies of bazel-toolchains."""
     excludes = native.existing_rules().keys()
 
     # ============================== Repositories ==============================
     if "io_bazel_rules_docker" not in excludes:
-        http_archive(
+        git_repository(
```

Comment:
Can we add a TODO or note that we should add the SHA check back eventually?

Reply:
Done